src/main/java/com/epam/ta/reportportal/ws/annotations/InValidator.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.ta.reportportal.ws.annotations;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public class InValidator implements ConstraintValidator<In, String> {
+
+	private String[] allowedValues;
+
+	@Override
+	public void initialize(In constraintAnnotation) {
+		allowedValues = constraintAnnotation.allowedValues();
+	}
+
+	@Override
+	public boolean isValid(String value, ConstraintValidatorContext context) {
+		if (null == value) {
+			return true;
```

Comment:
Do we want to have 'null' allowed with this annotation? 
If yes - we need to describe the 'null' values behavior in documentation

Reply:
Its already described in annotation docs